hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CommonConfigurationKeys.java

```diff
@@ -464,4 +464,40 @@
   /** Default value for IOStatistics logging level. */
   public static final String IOSTATISTICS_LOGGING_LEVEL_DEFAULT
       = IOSTATISTICS_LOGGING_LEVEL_DEBUG;
+
+  /**
```

Comment:
so, you mean move all "disk", "array", and "byte" options as well to azure? Or just the config and import this to Azure default values? Otherwise, we would need to copy all these for all stores implementing Data blocks